mediacapture-streams/historical.html

```diff
@@ -15,4 +15,8 @@
 test(function() {
   assert_false("mozGetUserMedia" in navigator);
 }, "navigator.mozGetUserMedia should not exist");
+
+test(() => {
+  assert_throws(new TypeError(), () => URL.createObjectURL(new MediaStream()));
```

Comment:
Can you move the creation of the `MediaStream` object out of the `assert_throws`?

Reply:
Yeah, that'd be good.